common-libs/common/src/Affinity.ts

```diff
@@ -241,6 +241,7 @@ export class Affinity {
         credential,
         documentLoader,
         verifyRevocationListCredential,
+        verifyMatchingIssuers: false,
```

Comment:
This change is required by breaking change in `vc-revocation-list` v3.
(`vc-revocation-list` v2 had some vulnerable dependencies)

Reply:
@inga-affinidi We have a revocation service: https://gitlab.com/affinidi/foundational/revocation-api, also have for the revocation service e2e/integration test at SDK. Please double check that revocation credentials issued with previous SDK version, still verifying OK using this updated one, if this the case, please go ahead and updated this dependency